test/functional/tools/creates_html.xml

```diff
@@ -0,0 +1,14 @@
+<tool id="creates_html" name="creates_html" version="1.0.0">
+  <command>
+    mkdir '$output.extra_files_path';
```

Comment:
I don't consider this to be best practice as of 16.04 - if tools specify a profile version of greater than ``16.01`` the commands run in an environment with ``set -e``. I've specified a tool version of 16.04 in a rebase to enable this. Can we call this point resolved or do you still insist on the ``CDATA``.